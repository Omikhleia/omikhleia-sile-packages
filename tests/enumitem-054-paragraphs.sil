\begin[papersize=a6, class=book]{document}
\script[src=packages/enumitem]
\font[family=Libertinus Serif]% Default Gentium lacks the white circle.
\nofolios
\neverindent
\set[parameter=document.parskip, value=5pt]%
\set[parameter=list.parskip, value=2pt]%
% Test case 054: both skips - alternate nesting

Text flow before.
\begin{enumerate}
    \item{Level 1 Enum
      \begin{itemize}
        \item{Level 2 Item
          \begin{enumerate}
            \item{Level 3 Enum}
          \end{enumerate}}
      \end{itemize}}
\end{enumerate}
Text flow after, just longer to be closer to the last list level.

Text flow before.

\begin{enumerate}
    \item{Level 1 Enum
      \begin{itemize}
        \item{Level 2 Item
          \begin{enumerate}
            \item{Level 3 Enum}
          \end{enumerate}
        Back to level 2 Item}
      \end{itemize}
    Back to level 1 Enum}
\end{enumerate}

Text flow after, just longer to be closer to the last list level.

\end{document}
