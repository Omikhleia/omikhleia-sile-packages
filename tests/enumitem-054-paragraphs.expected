Set paper size 	297.6377985	419.5275636
Begin page
Mx 	24.7039
My 	55.6452
Set font 	Libertinus Serif;10;400;;normal;;LTR
Mx 	5.3400
Mx 	4.4000
Mx 	4.9000
Mx 	3.1600
T	53 a=5.9700 70 a=4.4700 89 a=4.9000 85 a=3.1600	(Text)
Mx 	45.5242
Mx 	5.4000
Mx 	4.9700
Mx 	7.4700
T	2362 a=5.4000 80 a=5.0400 88 a=7.4700	(flow)
Mx 	66.3844
Mx 	5.0300
Mx 	4.4700
Mx 	3.1000
Mx 	5.0400
Mx 	3.6400
Mx 	4.3700
T	67 a=4.9300 70 a=4.4700 71 a=3.1000 80 a=5.0400 83 a=3.7200 70 a=4.4700	(before)
T	15 w=2.2000	(.)
Mx 	29.7039
My 	72.6452
T	18 w=4.6500	(1)
Mx 	34.3539
T	15 w=2.2000	(.)
Mx 	44.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	69.4033
T	18 w=4.6500	(1)
Mx 	77.0726
T	38 79 86 78 w=24.2000	(Enum)
Mx 	50.4489
My 	86.6452
T	1768 w=3.5100	(•)
Mx 	59.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	84.4021
T	19 w=4.6500	(2)
Mx 	92.0703
T	42 85 70 78 w=18.5000	(Item)
Mx 	64.7039
My 	100.6452
T	74 w=2.7100	(i)
Mx 	67.4139
T	15 w=2.2000	(.)
Mx 	79.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	104.3989
T	20 w=4.6500	(3)
Mx 	112.0639
T	38 79 86 78 w=24.2000	(Enum)
Mx 	24.7039
My 	117.6452
Mx 	5.3400
Mx 	4.4000
Mx 	4.9000
Mx 	3.1600
T	53 a=5.9700 70 a=4.4700 89 a=4.9000 85 a=3.1600	(Text)
Mx 	46.5289
Mx 	5.4000
Mx 	4.9700
Mx 	7.4700
T	2362 a=5.4000 80 a=5.0400 88 a=7.4700	(flow)
Mx 	68.3939
Mx 	4.5700
Mx 	5.9600
Mx 	4.4700
Mx 	3.1300
T	66 a=4.5700 2385 a=5.9600 70 a=4.4700 83 a=3.7200	(after)
T	13 w=2.2000	(,)
Mx 	92.7488
T	75 86 84 85 w=15.0900	(just)
Mx 	111.8638
T	77 80 79 72 70 83 w=26.2900	(longer)
Mx 	142.1787
T	85 80 w=8.2000	(to)
Mx 	154.4037
Mx 	5.0300
Mx 	4.4700
T	67 a=4.9300 70 a=4.4700	(be)
Mx 	167.9287
T	68 77 80 84 70 83 w=24.0500	(closer)
Mx 	196.0036
T	85 80 w=8.2000	(to)
Mx 	208.2286
T	85 73 70 w=13.0100	(the)
Mx 	225.2635
T	77 66 84 85 w=14.2700	(last)
Mx 	243.5585
T	77 74 84 85 w=12.4100	(list)
Mx 	24.7039
My 	129.6452
Mx 	2.6400
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	77 a=2.6400 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(level)
T	15 w=2.2000	(.)
Mx 	24.7039
My 	146.6452
Mx 	5.3400
Mx 	4.4000
Mx 	4.9000
Mx 	3.1600
T	53 a=5.9700 70 a=4.4700 89 a=4.9000 85 a=3.1600	(Text)
Mx 	45.5242
Mx 	5.4000
Mx 	4.9700
Mx 	7.4700
T	2362 a=5.4000 80 a=5.0400 88 a=7.4700	(flow)
Mx 	66.3844
Mx 	5.0300
Mx 	4.4700
Mx 	3.1000
Mx 	5.0400
Mx 	3.6400
Mx 	4.3700
T	67 a=4.9300 70 a=4.4700 71 a=3.1000 80 a=5.0400 83 a=3.7200 70 a=4.4700	(before)
T	15 w=2.2000	(.)
Mx 	29.7039
My 	163.6452
T	18 w=4.6500	(1)
Mx 	34.3539
T	15 w=2.2000	(.)
Mx 	44.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	69.4033
T	18 w=4.6500	(1)
Mx 	77.0726
T	38 79 86 78 w=24.2000	(Enum)
Mx 	50.4489
My 	177.6452
T	1768 w=3.5100	(•)
Mx 	59.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	84.4021
T	19 w=4.6500	(2)
Mx 	92.0703
T	42 85 70 78 w=18.5000	(Item)
Mx 	64.7039
My 	191.6452
T	74 w=2.7100	(i)
Mx 	67.4139
T	15 w=2.2000	(.)
Mx 	79.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	104.3989
T	20 w=4.6500	(3)
Mx 	112.0639
T	38 79 86 78 w=24.2000	(Enum)
Mx 	59.7039
My 	205.6452
T	35 66 68 76 w=19.8500	(Back)
Mx 	82.5682
T	85 80 w=8.2000	(to)
Mx 	93.7824
Mx 	2.6400
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	77 a=2.6400 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(level)
Mx 	115.8367
T	19 w=4.6500	(2)
Mx 	123.5009
T	42 85 70 78 w=18.5000	(Item)
Mx 	44.7039
My 	219.6452
T	35 66 68 76 w=19.8500	(Back)
Mx 	67.5693
T	85 80 w=8.2000	(to)
Mx 	78.7848
Mx 	2.6400
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	77 a=2.6400 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(level)
Mx 	100.8402
T	18 w=4.6500	(1)
Mx 	108.5056
T	38 79 86 78 w=24.2000	(Enum)
Mx 	24.7039
My 	236.6452
Mx 	5.3400
Mx 	4.4000
Mx 	4.9000
Mx 	3.1600
T	53 a=5.9700 70 a=4.4700 89 a=4.9000 85 a=3.1600	(Text)
Mx 	46.5289
Mx 	5.4000
Mx 	4.9700
Mx 	7.4700
T	2362 a=5.4000 80 a=5.0400 88 a=7.4700	(flow)
Mx 	68.3939
Mx 	4.5700
Mx 	5.9600
Mx 	4.4700
Mx 	3.1300
T	66 a=4.5700 2385 a=5.9600 70 a=4.4700 83 a=3.7200	(after)
T	13 w=2.2000	(,)
Mx 	92.7488
T	75 86 84 85 w=15.0900	(just)
Mx 	111.8638
T	77 80 79 72 70 83 w=26.2900	(longer)
Mx 	142.1787
T	85 80 w=8.2000	(to)
Mx 	154.4037
Mx 	5.0300
Mx 	4.4700
T	67 a=4.9300 70 a=4.4700	(be)
Mx 	167.9287
T	68 77 80 84 70 83 w=24.0500	(closer)
Mx 	196.0036
T	85 80 w=8.2000	(to)
Mx 	208.2286
T	85 73 70 w=13.0100	(the)
Mx 	225.2635
T	77 66 84 85 w=14.2700	(last)
Mx 	243.5585
T	77 74 84 85 w=12.4100	(list)
Mx 	24.7039
My 	248.6452
Mx 	2.6400
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	77 a=2.6400 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(level)
T	15 w=2.2000	(.)
End page
Finish
